sdk/identity/identity-broker/samples/v1-beta/typescript/README.md

```diff
@@ -0,0 +1,80 @@
+---
+page_type: sample
+languages:
+  - typescript
+products:
+  - azure
+  - azure-active-directory
+  - WAM
+  - MSA support
+urlFragment: identity-broker-typescript-beta
+---
+
+# Azure Identity Brokered Auth Plugin client library samples for TypeScript (Beta)
+
+These sample programs show how to use the TypeScript client libraries for Azure Identity Brokered Auth Plugin in some common scenarios.
+
+| **File Name**     | **Description**       |
+| ----------------- | --------------------- |
+| [index.ts][index] | Demonstrates usage of |
```

Comment:
Need to finish writing the description here too

Reply:
done